docs/content/en/docs/reference/clusterspec/optional/irsa.md

```diff
@@ -83,41 +110,65 @@ Set the remaining fields in [cluster spec]({{< relref "../vsphere/" >}}) as requ
 1. The cluster provisioning workflow generates a pair of service account signing keys. Retrieve the public signing key generated and used by the cluster, and create a keys.json document containing the public signing key. 
 
     ```bash
-    kubectl get secret ${CLUSTER_NAME}-sa -n eksa-system -o jsonpath={.data.tls\\.crt} | base64 --decode > ${CLUSTER_NAME}-sa.pub
-    wget https://raw.githubusercontent.com/aws/amazon-eks-pod-identity-webhook/master/hack/self-hosted/main.go -O keygenerator.go
-    go run keygenerator.go -key ${CLUSTER_NAME}-sa.pub | jq '.keys += [.keys[0]] | .keys[1].kid = ""' > keys.json
+    git clone https://github.com/aws/amazon-eks-pod-identity-webhook
+    cd amazon-eks-pod-identity-webhook
+    kubectl get secret ${CLUSTER_NAME}-sa -n eksa-system -o jsonpath={.data.tls\\.crt} | base64 --decode > ${CLUSTER_NAME}-sa.pub    
+    go run ./hack/self-hosted/main.go -key ${CLUSTER_NAME}-sa.pub | jq '.keys += [.keys[0]] | .keys[1].kid = ""' > keys.json
     ```
-1. Upload the keys.json document to the s3 bucket.
+2. Upload the keys.json document to the s3 bucket.
     ```bash
     aws s3 cp --acl public-read ./keys.json s3://$S3_BUCKET/keys.json
     ```
 
-#### Deploy pod identity webhook
+### Deploy pod identity webhook
 
 1. After hosting the service account public signing key and OIDC discovery documents, the applications running in pods can start accessing the desired AWS resources, as long as the pod is mounted with the right service account tokens. This part of configuring the pods with the right service account tokens and env vars is automated by the [amazon pod identity webhook](https://github.com/aws/amazon-eks-pod-identity-webhook). Once the webhook is deployed, it mutates any pods launched using service accounts annotated with `eks.amazonaws.com/role-arn`
 
-1. Check out [this commit](https://github.com/aws/amazon-eks-pod-identity-webhook/commit/a65cc3d9c61cf6fc43f0f985818c474e0867d786) of the amazon-eks-pod-identity-webhook.
+1. Clone [amazon-eks-pod-identity-webhook](https://github.com/aws/amazon-eks-pod-identity-webhook) if not done already.
+
+2. Set the $KUBECONFIG env var to the path of the EKS Anywhere cluster.
```

Comment:
Change all these steps to "1." all the indents look fine.

Reply:
Thanks for your observations. I've updated and committed the changes.